bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/OperationAccountant.java

```diff
@@ -58,6 +61,7 @@
 
   private long noSuccessCheckDeadlineNanos;
   private int noSuccessWarningCount;
+  private int idGenerator = 0;
```

Comment:
nit: Seems weird to call an int a "generator". Also why an int guarded by a lock vs an atomic like above for the batch generator?